### PR TITLE
github pages added

### PR DESCRIPTION
**Pull request by claireolmstead**
Purpose
---------------
Publish the site so that people can test it and view progress without having to run it locally.

Solution
---------------
deploy with github pages

Change summary
---------------
* add gh-pages package
* add homepage and build instructions
* config pages to look at the gh-pages branch

Steps to Verify
----------------
1. Make sure the link works.

Additional details / screenshot
----------------
link to the published site:
https://libertydsnp.github.io/example-client/

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
+.Login__block .ant-btn {
```
Comment (Ajaxan) [inline src/App.css:1]: Remove .css files.

File: package.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "client",
+  "homepage": "http://LibertyDSNP.github.io/example-client",
```
Comment (Ajaxan) [inline package.json:3]: We still need a blockchain backend for this to work. Currently we've just been using the old demo contracts repo, but we can probably work on something new. Perhaps we can even include that in the repo? We should probably discuss it next week.

**Review by Ajaxan — CHANGES_REQUESTED** [review]
We will need to setup a blockchain for the example client to fully get this up and working.

File: package.json [hunk 3]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "client",
+  "homepage": "https://libertydsnp.github.io/example-client/",
```
Comment (wilwade) [inline package.json:3]: This is just me, but I think the home page should still be the github repo.

**Pull request closed without merging**